src/main/java/view_post/interface_adapters/ViewPostPresenter.java

```diff
@@ -1,4 +1,31 @@
 package view_post.interface_adapters;
 
-public class ViewPostPresenter {
+import view_post.use_case.ViewPostOutputBoundary;
+import view_post.use_case.ViewPostResponseModel;
+
+import java.util.ArrayList;
+
+public class ViewPostPresenter implements ViewPostOutputBoundary {
+    private ViewPostViewModel viewModel;
+
+    public ViewPostPresenter(ViewPostViewModel viewModel){
+        this.viewModel = viewModel;
+    }
+    @Override
+    public void updateActivePost(ViewPostResponseModel responseModel) {
+        String postEmail = responseModel.getPosterEmail();
+        String postBody = responseModel.getPostBody();
+        String postTags = responseModel.getPostTags();
+        ArrayList<Integer> postReplies = responseModel.getPostReplies();
+        String deadline = responseModel.getDeadline();
+        String creationDate = responseModel.getCreationDate();
+        String collaborators = responseModel.getCollaborators();
+        int postID = responseModel.getPostID();
+        String title = responseModel.getTitle();
+
+        ViewPostOutputData outputData = new ViewPostOutputData(postEmail, postBody, postTags, postReplies, deadline,
```

Comment:
The GitHub auto grading is not finding the ViewPostOutputData class and I can't find it in your branch either. Did you forget to upload it or is it a work in progress?

Reply:
GOOD CATCH it wasn't added to Git for some reason. I'm gonna push right now.

Reply:
Great!